docker/keygenerator/Dockerfile

```diff
@@ -13,4 +13,4 @@ FROM ubuntu:22.04
 COPY --from=builder /go/mx-chain-go/cmd/keygenerator /go/mx-chain-go/cmd/keygenerator
 
 WORKDIR /go/mx-chain-go/cmd/keygenerator/
-ENTRYPOINT ["./keygenerator"]
+ENTRYPOINT ["/go/mx-chain-go/cmd/keygenerator/keygenerator"]
```

Comment:
Or maybe `WORKDIR /go/mx-chain-go/cmd/keygenerator` (without the trailing slash)?

Reply:
No longer applicable.